cmd/contour/ingressstatus.go

```diff
@@ -97,3 +98,26 @@ func (isw *loadBalancerStatusWriter) Start(stop <-chan struct{}) error {
 		}
 	}
 }
+
+func parseStatusFlag(status string) v1.LoadBalancerStatus {
```

Comment:
Add a simple test for the three cases here?

Reply:
There's only two cases? But I can add a test.

Reply:
I was thinking IPv4, IPv4, arbitrary string as the 3 cases.

Reply:
IPv6, IPv4, arbitrary string makes sense, sure.